pkgs/development/python-modules/stytra/default.nix

```diff
@@ -79,6 +79,9 @@ buildPythonPackage rec {
     pyserial
   ];
 
+  # module 'pyqtgraph' has no attribute 'GraphicsWindow'
```

Comment:
Is this just a problem of the tests or is the library incompatible?

Reply:
I really have no idea unfortunately.

Reply:
Found the issue, see 2f1773b57be96260cc8decc4e37f8ff47a779359